FabricExample/e2e/examplesTests/tests.e2e.ts

```diff
@@ -0,0 +1,111 @@
+import { device, expect, element, by } from 'detox';
+
+describe('Test432', () => {
+  beforeAll(async () => {
+    await device.reloadReactNative();
+  });
+
+  it('should Test432 exist', async () => {
+    await waitFor(element(by.id('root-screen-tests-Test432')))
+    .toBeVisible()
+    .whileElement(by.id('root-screen-examples-scrollview'))
+    .scroll(600, 'down', NaN, 0.85);
+
+    await expect(element(by.id('root-screen-tests-Test432'))).toBeVisible();
+    await element(by.id('root-screen-tests-Test432')).tap();
+  });
+
+  it('home-square should be fully visible', async () => {
+    await expect(element(by.id('home-square'))).toBeVisible(100);
+  });
+
+  it('squares from details screen should be fully visible', async () => {
+    await element(by.id('go-to-details')).tap();
+    await expect(element(by.id('details-red-square'))).toBeVisible(100);
+
+    await element(by.id('details-toggle-subviews')).tap();
```

Comment:
```suggestion
    await element(by.id('details-button-toggle-subviews')).tap();
```

As for assigning testID I do also think that we should have a clear convention. Open for discussion there. I would suggest: `<where>-<role-of-the-element>-<action>`

* `where` might be relative to the *test* root screen (not app)
* `role-of-the-element` I mean button / view / header item / header left / etc.
* `action` - in case there is any, e.g. what the button click does (if can be put in 2-4 words, otherwise some generic phrase).

Reply:
This applies widely to this PR.

Reply:
Makes sense as well.

Reply:
d2afd9a8d2acb7e43f6ad65571025287402cb770